pkg/tuf/policy.go

```diff
@@ -0,0 +1,208 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Contains root policy definitions.
+// Eventually, this will move this to go-tuf definitions.
+
+package tuf
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+
+	cjson "github.com/secure-systems-lab/go-securesystemslib/cjson"
+)
+
+type Signed struct {
```

Comment:
I'm fine with adding comments later, but if they could be done immediately in another PR, that'd be great so we don't leave tech debt.

Reply:
I'm not sure I'm the person you want writing doc comments for these, is the only problem 😆 

Could I file an issue (on whom I don't know) to followup and add these comments?

Reply:
I can take care of it!